src/js/snapshot.ts

```diff
@@ -12,6 +12,7 @@ export function getExpectedKeys() {
     API,
     scheduleCallback,
     API,
+    {},
```

Comment:
Confusing, this just undoes my change? You have any idea which of your subsequent changes made it go back? We probably need to see if we can come up with a better way to keep this list up to date...

Reply:
> Confusing, this just undoes my change? You have any idea which of your subsequent changes made it go back? We probably need to see if we can come up with a better way to keep this list up to date...

I think changing `typeof globalThis.API` ==> `typeof API` made it go back for some reason... I have no idea way.